src/Sarif/Baseline/V2/WhatComparer.cs

```diff
@@ -33,7 +33,7 @@ public static IEnumerable<WhatComponent> WhatProperties(this ExtractedResult res
             {
                 foreach (var fingerprint in result.Result.Fingerprints)
                 {
-                    yield return new WhatComponent(result.RuleId, "Fingerprint", fingerprint.Key, fingerprint.Value);
+                    yield return new WhatComponent(result.RuleId, "Fingerprint", "Fingerprint/" + fingerprint.Key, fingerprint.Value);
```

Comment:
>Key [](start = 110, length = 3)

In principle, elements of the `Fingerprints`, `PartialFingerprints`, and `Properties` dictionaries might have the same keys, so we disambiguate them.